packages/cli/src/commands/fix/tasks/initialize-task.ts

```diff
@@ -45,7 +45,7 @@ export function initTask(
         projectType = 'glimmer';
       }
 
-      preFlightCheck(rootPath, projectType);
+      preFlightCheck(rootPath, projectType, skipDepsCheck, skipEsLintCheck);
```

Comment:
lets change the last param to a record if we need to expand to include additional `skips`

```
const skipChecks = {
skipDepsCheck,
skipEsLintCheck
}
```

eg. `preFlightCheck(rootPath, projectType, skipChecks)`

Reply:
sure thing